packages/table/src/table.tsx

```diff
@@ -450,6 +450,18 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         );
     }
 
+    public resizeRowsBasedOnColumnHeight(columnIndex: number) {
```

Comment:
@gscshoyru when glancing at this API, I wasn't really sure what "based on column height" meant ("aren't all the columns the same height / isn't that how tables work?"). Is this the best name for this function?

More verbose but more clear to me: `resizeRowsBasedOnTallestVisibleCellInColumn`